suite/package.json

```diff
@@ -39,5 +41,6 @@
     "ts-jest": "^29.1.1",
     "typescript": "^4.9.5"
   },
+  "prettier": "eslint-config-3box/prettier.config",
```

Comment:
Does this mean you can run `npm prettier` to run it? Or do we need an explicit `script` entry for the prettier?

Reply:
We would need to add an explicit script entry.  I can do so if you'd like, we have such a script already in the js-ceramic repo I can just copy.  I usually just rely on my IDE to run prettier on save though

Reply:
this line is providing prettier the configuration we want it to use when it runs.